ACPs/194-streaming-asynchronous-execution/README.md

```diff
@@ -153,21 +162,20 @@ Other than the _earliest_ (genesis) named block, which MUST be interpreted in th
 > The finality guarantees of Snowman consensus remove any distinction between _safe_ and _finalized_. 
 > Furthermore, the _latest_ block is not at risk of re-org, only of a negligible risk of data corruption local to the API node.
 
-
-### Optimal block building
-
-As EOA-to-EOA transfers of value are entirely guaranteed upon _acceptance_, block builders SHOULD prioritise other transactions for earlier execution.
-A reliable marker of such transactions is a gas limit of 21,000 as this is an indication from the sender that they do not intend to execute bytecode.
-
-Checking for code at the destination address is both unnecessary and unreliable as the address might be that of a contract created earlier in the queue.
-
 ## Backwards Compatibility
 
-All ACPs that introduce backwards incompatibilities must include a section describing these incompatibilities and their severity. The ACP should provide recommendations for dealing with these incompatibilities.
+This ACP modifies the meaning of multiple fields in the block. A comprehensive list of changes will be produced once a reference implementation is available.
+
+Likely fields to change include:
+- `stateRoot`
+- `receiptsRoot`
+- `logsBloom`
+- `gasUsed`
```

Comment:
(fyi) I've thus far kept `gasLimit` unchanged (i.e. the txs being pushed to the queue), which adds an interesting possibility of `gasUsed > gasLimit` because they refer to different tranches of transactions.